mat64/triangular_test.go

```diff
@@ -32,46 +33,98 @@ func (s *S) TestNewTriangular(c *check.C) {
 			}},
 		},
 	} {
-		t := NewTriDense(test.N, test.upper, test.data)
-		rows, cols := t.Dims()
-		c.Check(rows, check.Equals, test.N, check.Commentf("Test %d", i))
-		c.Check(cols, check.Equals, test.N, check.Commentf("Test %d", i))
-		c.Check(t, check.DeepEquals, test.mat, check.Commentf("Test %d", i))
+		tri := NewTriDense(test.n, test.upper, test.data)
+		rows, cols := tri.Dims()
+
+		if rows != test.n {
+			c.Errorf("unexpected number of rows for test %d: got: %d want: %d", i, rows, test.n)
```

Comment:
I'm not sure what you are asking for here. do you want {got,want}{Rows,Cols} and {got,want}Tri? I don't think that makes the code any easier to read.